tests/phpBackend/cachedClient/phpxmlrpcValidator1EasyStructTestCacheOffByDefault1.phpt

```diff
@@ -22,7 +22,7 @@ $options = array(
     'cacheDebug' => true
 );
 
-$client = XML_RPC2_CachedClient::create('http://phpxmlrpc.sourceforge.net/server.php', $options);
+$client = XML_RPC2_CachedClient::create('https://gggeek.altervista.org/sw/xmlrpc/demo/server/server.php', $options);
```

Comment:
Go to the original link and you get redirected to the second, which causes the test to fail.